src/components/_classes/component/editForm/utils.js

```diff
@@ -12,12 +12,18 @@ const EditFormUtils = {
       if (objValue.key === srcValue.key) {
         // Create complete objects by including missing keys.
         _.each(objValue, (value, prop) => {
-          if (!srcValue.hasOwnProperty(prop)) {
+          if (objValue.overrideEditForm) {
+            srcValue[prop] = value;
+          }
+          else if (!srcValue.hasOwnProperty(prop)) {
             srcValue[prop] = value;
           }
         });
         _.each(srcValue, (value, prop) => {
-          if (!objValue.hasOwnProperty(prop)) {
+          if (srcValue.overrideEditForm) {
+            srcValue[prop] = value;
```

Comment:
Shouldn't this say...

```js
if (srcValue.overrideEditForm) {
  objValue[prop] = value;
}
```

That being said, couldn't this be simplified with.

```js
_.each(objValue, (value, prop) => {
  if (objValue.overrideEditForm || !srcValue.hasOwnProperty(prop)) {
    srcValue[prop] = value;
  }
});
_.each(srcValue, (value, prop) => {
  if (srcValue.overrideEditForm || !objValue.hasOwnProperty(prop)) {
    objValue[prop] = value;
  }
});
```

Reply:
done